internal/adapters/operation/operation_storage_redis.go

```diff
@@ -217,22 +215,29 @@ func (r *redisOperationStorage) ListSchedulerActiveOperations(ctx context.Contex
 	return operations, nil
 }
 
-func (r *redisOperationStorage) ListSchedulerFinishedOperations(ctx context.Context, schedulerName string) (operations []*operation.Operation, err error) {
-	currentTime := r.clock.Now()
-	lastDay := r.clock.Now().Add(-24 * time.Hour)
-	operationsIDs, err := r.getFinishedOperationsFromHistory(ctx, schedulerName, lastDay, currentTime)
-	var nonExistentOperations []string
-	var executions = make(map[string]redis.Cmder)
+func (r *redisOperationStorage) ListSchedulerFinishedOperations(ctx context.Context, schedulerName string, page, pageSize int64) (operations []*operation.Operation, total int64, err error) {
+	if err := r.CleanExpiredOperations(ctx, schedulerName); err != nil {
+		return nil, 0, fmt.Errorf("failed to clean scheduler expired operations: %w", err)
```

Comment:
Hey @mflilian when errors occur normally integer fields are filled with invalid numbers semantically saying. So what do you thing on the errors to put numbers less than zero to symbolize an invalid `page`?

Reply:
It makes sense, @arthur29 ! Just applied your suggestions